api/src/modules/Session/model.ts

```diff
@@ -36,6 +41,40 @@ export const SessionCreateOutput = objectType({
   },
 });
 
+// Actions
+
+function validateOrigin(originParam: string): void {
+  if (originParam === '*') {
+    return;
+  }
+  const trimmedOriginParam = originParam.trim();
+  const origins = trimmedOriginParam.split(',');
```

Comment:
This will also trim spaces between origins

```suggestion
  const trimmedOriginParam = originParam.trim();
  const origins = trimmedOriginParam.split(',').map(origin => origin.trim());
```

But since you trim it here, I would call it `parseOriginMarkup` and return cleanedup value which is then stored in the database